examples/mnist.cpp

```diff
@@ -245,6 +245,7 @@ void testMNISTLoadAndTraining() {
   const char *inputName = "data";
 
   llvm::Error errPtr = llvm::Error::success();
+  MARK_ERR_CHECKED(errPtr);
```

Comment:
Do you think it would be clearer/more concise to have a MAKE_SUCCESS() that magicks up a checked `success()` value?  (Would that even work?)  Every where we use `MARK_ERR_CHECKED` it seems like it's because we're creating an `Error` to be possibly filled in later.

Reply:
I was thinking about how to do this but I can't think of one, can you?